wrangler-transform/src/main/java/io/cdap/wrangler/Precondition.java

```diff
@@ -45,14 +49,28 @@ public Precondition(String condition) throws PreconditionException {
     }
   }
 
+  public ScriptContext createContext() {
```

Comment:
Not familiar with ScriptConext, I saw this part is copied from `CompiledScript.eval`, is it implementation specific ? will it work with later java versions ?

Reply:
It does not use any private API, so I don't expect problems here

Reply:
Also it's not from specific implementation, but from CompiledScript itself